package.json

```diff
@@ -9,23 +9,23 @@
   ],
   "types": "./lib/src/index.d.ts",
   "devDependencies": {
-    "@types/superagent": "^2.0.36",
     "@types/mocha": "^2.2.34",
+    "@types/node": "^9.4.0",
```

Comment:
Cannot successfully build this library without node typings for require statements